Python/Product/DebuggerHelper/trace.cpp

```diff
@@ -51,10 +51,10 @@ struct {
         int64_t ob_size;
     } PyVarObject;
     struct {
-        int64_t f_back, f_code, f_globals, f_locals, f_lineno;
+        int64_t f_back, f_code, f_globals, f_locals, f_lineno, f_frame;
     } PyFrameObject;
     struct {
-        int64_t co_varnames, co_filename, co_name;
```

Comment:
co_varnames wasn't used thankfully in the trace code. So I just removed it. It doesn't exist anymore in 3.11 and would have been a pain to add support for here.